iset.mm

```diff
@@ -142684,6 +142684,79 @@ S C_ ( P ( ball ` D ) T ) ) $=
+###############################################################################
+$)
+
+
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Basic trigonometry
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  The exponential, sine, and cosine functions (cont.)
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $( The exponential function is continuous.  (Contributed by Paul Chapman,
+     15-Sep-2007.)  (Revised by Mario Carneiro, 20-Jun-2015.) $)
+  efcn $p |- exp e. ( CC -cn-> CC ) $=
+    ( cc wss ce wf ccncf co wcel ssid eff w3a cdv cdm wceq dvef feq1i fdmi dvcn
+    mpbir mpan2 mp3an ) AABZAACDZUACAAEFGZAHZIUDUAUBUAJACKFZLAMUCAAUEAAUEDUBIAA
+    UECNORPAACQST $.
+
+  ${
+    $d x y $.
+    $( Sine is continuous.  (Contributed by Paul Chapman, 28-Nov-2007.)
+       (Revised by Mario Carneiro, 3-Sep-2014.) $)
+    sincn $p |- sin e. ( CC -cn-> CC ) $=
+      ( vx vy cc ci cv cmul co cfv cmin cdiv cmpt wcel wtru ccom eqid mulc1cncf
+      ce a1i mp1i cncfmpt1f csin cneg c2 ccncf df-sin wral cmopn ctx subcncntop
+      wf cabs ccn efcn ax-icn negicn cncfmpt2fcntop cncff syl fmpt sylibr eqidd
+      oveq1 fmptcof cc0 cap wbr 2muliap0 divccncfap mp2an cncfco eqeltrrd mptru
+      2mulicn eqeltri ) UAACDAEZFGZQHZDUBZVOFGZQHZIGZUCDFGZJGZKZCCUDGZAUEWDWELM
+      BCBEZWBJGZKZACWAKZNWDWEMABCCWAWGWCWIWHMCCWIUJZWACLACUFMWIWELWJMAVQVTIUKIN
+      UGHZCWKOZIWKWKUHGWKULGLMWKWLUIRMAVPQCQWELMUMRZDCLACVPKZWELMUNADWNWNOPSTMA
+      VSQCWMVRCLACVSKZWELMUOAVRWOWOOPSTUPZCCWIUQURACCWAWIWIOUSUTMWIVAMWHVAWFWAW
+      BJVBVCMCCCWIWHWPWHWELZMWBCLWBVDVEVFWQVMVGBWBWHWHOVHVIRVJVKVLVN $.
+
+    $( Cosine is continuous.  (Contributed by Paul Chapman, 28-Nov-2007.)
+       (Revised by Mario Carneiro, 3-Sep-2014.) $)
+    coscn $p |- cos e. ( CC -cn-> CC ) $=
+      ( vx vy cc ci cv cmul co ce caddc cdiv cmpt wcel wtru ccom eqid mulc1cncf
+      cfv c2 a1i mp1i ccos cneg ccncf df-cos wf wral cabs cmin cmopn addcncntop
+      ctx ccn efcn ax-icn cncfmpt1f negicn cncfmpt2fcntop cncff syl fmpt sylibr
+      eqidd oveq1 fmptcof cc0 cap wbr 2cn 2ap0 divccncfap mp2an cncfco eqeltrrd
+      mptru eqeltri ) UAACDAEZFGZHQZDUBZVPFGZHQZIGZRJGZKZCCUCGZAUDWDWELMBCBEZRJ
+      GZKZACWBKZNWDWEMABCCWBWGWCWIWHMCCWIUEZWBCLACUFMWIWELWJMAVRWAIUGUHNUIQZCWK
+      OZIWKWKUKGWKULGLMWKWLUJSMAVQHCHWELMUMSZDCLACVQKZWELMUNADWNWNOPTUOMAVTHCWM
+      VSCLACVTKZWELMUPAVSWOWOOPTUOUQZCCWIURUSACCWBWIWIOUTVAMWIVBMWHVBWFWBRJVCVD
+      MCCCWIWHWPWHWELZMRCLRVEVFVGWQVHVIBRWHWHOVJVKSVLVMVNVO $.
+  $}
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Properties of pi = 3.14159...
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $( Lemma for pire , pigt2lt4 and sinpi .  (Contributed by Mario Carneiro,
```

Comment:
do not forget to insert "~" here to reference the mentioned theorems as soon as they are available.